### DISCUSSION TITLE
What should be the default behavior when running `docker run zfnd/zebra`

### DISCUSSION THREAD

**Discussion by gustavovalverde** [ZcashFoundation/zebra#5972] [Brainstorming]
While working with https://github.com/ZcashFoundation/zebra/pull/5968 I realized our Docker `CMD` and `ENTRYPOINT` might not have an optimal logic.

Nonetheless, to change this we should agree on what the default behavior should be when someone runs `docker run zfnd/zebra`, as of now the default is running `zebrad -c <configuration_file_path> start`, using the configuration file built in the Dockerfile: https://github.com/ZcashFoundation/zebra/blob/main/docker/Dockerfile#L165-L180

Should we expect something different?

**Comment by teor2345** [comment]
I think the Dockerfile config is fine, the default Zebra config would probably also be fine.

I'm not sure about the Zebra state path in the Dockerfile config. It's not what Zebra would use by default outside Docker, or if run inside Docker without a config. What is the convention for cache or permanent data paths in Docker images?

**Comment by mpguerra** [comment]
@gustavovalverde did you get your answer here, can you close this one if so?
